src/payments/p2pkh.js

```diff
@@ -27,7 +27,10 @@ function p2pkh(a, opts) {
     a,
   );
   const _address = lazy.value(() => {
-    const payload = Buffer.from(bs58check.decode(a.address));
+    const bs58DecodeFunc = !a.bs58DecodeFunc
+      ? bs58check.decode
+      : a.bs58DecodeFunc;
```

Comment:
Are we allowed to use `??` in here? Otherwise, would `!= null` make this cleaner?

Similarly for the other 4 place this pattern exists in this commit.

Reply:
The reason is that an older version of node is used (version 12) and no transpiler like babel is used to target the older version. So the JS artifacts will include `??` in them causing errors by the cjs module loader. I can upgrade from Node 12 to 16, but that would cause a new package-lock version and format the whole package-lock. I think this should happen upstream first to make our lives easier to in the future when we need to pull changes from upstream.